CorsixTH/Lua/languages/czech.lua

```diff
@@ -1143,6 +1143,7 @@ tooltip = {
 	shift_scrollspeed = "Nastavte rychlost rolování s shiftem v rozmezí 1 (nejpomalejší) až 10 (nejrychlejší). Výchozí hodnota je 4.",
 	zoom_speed = "Nastavte rychlost přibližování v rozmezí 10 (nejpomalejší) až 1000 (nejrychlejší). Výchozí hodnota je 80.",
 	cancel_zoomspeed = "Neměnit rychlost přibližování.",
+        check_for_updates = "Nastavuje, zda má hra při spuštění kontrolovat aktualizace.",
     },
```

Comment:
There's a general alignment issue here caused by a random 4 space indent, and can't seem to resolve it my self. I can try and fix it in a separate PR rather than here?

Reply:
Elsewhere is a good idea I think, and you're fighting TABs versus spaces:

Tabs are visible in my editor, and it shows
![Screenshot from 2022-10-26 08-07-16](https://user-images.githubusercontent.com/3254232/197947896-4d573216-852e-4c6b-85bf-cf27de0ab51e.png)